spec/teamsnap_init_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "teamsnap"
+
+RSpec.describe "teamsnap_rb" do
```

Comment:
As an experiment, I'd like to move this entire file into the existing `teamsnap_spec.rb`. So, we'd have two `RSpec.describe` blocks in the file.